### ISSUE TITLE
support empty comma seperated arguments and parameters

### ISSUE THREAD

**Issue by plexigras** [wcjohnson/lightscript#78]
just so `x(, b) => y(, b)` compile to `const x = (arg0, b) => y(undefined, b)`

**Comment by wcjohnson** [comment]
Yes, I think this is a good idea at broad strokes, but I would like to bikeshed the syntax a bit before implementing it.

For example, a double comma `,,` is a very common typo -- I don't know if it's a good idea for that to compile and produce difficult-to-diagnose runtime crashes.

Perhaps a whitespace should be required between the omitted arg list entries.

**Comment by plexigras** [comment]
I didn't think about it potentially being a typo but now that I think about it I think it might be better if eslint would warn you about the potential typo over having it directly in the transpiler.
